### PR TITLE
Momza 2292 sms registration

### PR DESCRIPTION
**Pull request by Hlamallama**

### CODE REVIEW

File: ndoh_hub/urls.py [hunk 1]
```diff
@@ -97,4 +99,9 @@
     path(
         "metrics", internal_only(django_prometheus.ExportToDjangoView), name="metrics"
     ),
+    re_path(
+        r"^api/v1/send-whatsapp-template",
```
Comment (erikh360) [inline ndoh_hub/urls.py]: We usually don't have '-' in paths

File: ndoh_hub/serializers.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from rest_framework import serializers
+
+
+class WhatsappTemplateMessageSerializer(serializers.Serializer):
+    class ParametersField(serializers.Serializer):
```
Comment (erikh360) [inline ndoh_hub/serializers.py:5]: Please add some tests for invalid requests

**Review by erikh360 — COMMENTED** [review]
Just the two small comments

What is the test_decorators file for and why was it renamed?

**Comment by Hlamallama** [comment]
> Just the two small comments
> 
> What is the test_decorators file for and why was it renamed?

Moved it and test_utils into a tests folder

**Review by erikh360 — APPROVED** [review]

**Pull request merged**